src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -117,6 +117,13 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="XA0107" xml:space="preserve">
+    <value>{0} is a Reference Assembly.</value>
+  </data>
+  <data name="XA0107_Ignoring" xml:space="preserve">
+    <value>Ignoring Reference Assembly `{0}`.</value>
+    <comment>{0} - File name of the assembly.</comment>
+  </data>
```

Comment:
I debated on whether to split XA0107 into two different codes for the two different messages. For the moment, I left them both under one code. In contrast to XA3001, which I split into separate codes as part of https://github.com/xamarin/xamarin-android/commit/1524e6e24780de610faa299065285ef63b790fef, the two messages for XA0107 have the same underlying cause. The reason there are two messages for XA0107 is instead that the messages are emitted from different places. The meaning of the warning differs slightly in those different places.

If telemetry some day indicates that XA0107 is appearing frequently in user projects and the team would like to know which of the code paths is most frequently causing the warning, new message codes can be added at that time. (Note that `XA0107_Ignoring` is currently used in two different places, so having separate codes for each of the two messages wouldn't actually be enough to distinguish all of the different code paths anyway.)

Reply:
Further note that the two "different yet related" messages exist so that things "look reasonable" within Visual Studio, as multi-line messages get rendered in the Warning/Error panel "weirdly".